### ISSUE TITLE
New models

### ISSUE THREAD

**Issue by MarcCoru** [maxzoll/BreizhCrops#1]
Hi Max,
I just added the inceptiontime model in the upstream repo

hyperparameter are `hidden_dims` and `num_layers`
see 

```
        model = InceptionTime(input_dim=ndims, num_classes=num_classes,
                              num_layers=args.hyperparameter["num_layers"],
                              hidden_dims=args.hyperparameter["hidden_dims"], device=device).to(device)
```

Also, I replaced the transformer implementation with the new one provided by pytorch (makes the code much shorter!)

The hyperparameters are: `hidden_dims`, `n_head`, `num_layers`,`relu`,`dropout`
see
```
        model = Transformer(input_dim=ndims, num_classes=num_classes, sequencelength=sequencelength,
                            d_model=args.hyperparameter["hidden_dims"],
                            n_head=args.hyperparameter["n_head"],
                            n_layers=args.hyperparameter["num_layers"],
                            d_inner=4*args.hyperparameter["hidden_dims"],
                            activation="relu",
                            dropout=args.hyperparameter["dropout"]
                            ).to(device)
```

I developed the code until training started on my side. Do you think you could add these to the hyperparameter tuning as well?

**Comment by MarcCoru** [comment]
Also added StarRNN implementation from Türkoglu Mehmet Özgür <ozgur.turkoglu@geod.baug.ethz.ch>

```
model = StarRNN(input_dim=ndims,
                        num_classes=num_classes,
                        hidden_dims=args.hyperparameter["hidden_dims"],
                        num_layers=args.hyperparameter["num_layers"],
                        dropout=args.hyperparameter["dropout"],
                        bidirectional=False,
                        use_batchnorm=False,
                        use_layernorm=True,
                        device=device).to(device)
```
